lib/protoflow-core/src/comparable_any.rs

```diff
@@ -0,0 +1,87 @@
+// This is free and unencumbered software released into the public domain.
+
+use core::{
+    cmp::Ordering,
+    ops::{Deref, DerefMut},
+};
+
+use crate::{prelude::prost_types::Any, Message};
+
+#[derive(Debug, Clone, Default)]
+pub struct ComparableAny(pub Any);
+
+impl PartialOrd for ComparableAny {
+    fn partial_cmp(&self, other: &Self) -> Option<Ordering> {
+        Some(self.0.type_url.cmp(&other.0.type_url))
```

Comment:
Is the intention to *only* sort based on the type url? Should this not compare on `type_url` and if equal, `value`?
Maybe if should even be `None` if `self.0.type_url != other.0.type_url` and `Some(Ordering)` when the type_urls match, not sure.

Reply:
fixed it, it was a bug, values should have been compared.